.kokoro/tests/run_tests.sh

```diff
@@ -75,16 +75,18 @@ if [[ "$SCRIPT_DEBUG" != "true" ]]; then
     "java-firestore-samples-secrets.txt" \
     "java-cts-v4-samples-secrets.txt" \
     "java-cloud-sql-samples-secrets.txt" \
+    "java-iam-samples-secrets.txt" \
     "java-scc-samples-secrets.txt")
 
     # create secret dir
     mkdir -p "${KOKORO_GFILE_DIR}/secrets"
-    
+
     for SECRET in "${SECRET_FILES[@]}"; do
       # grab latest version of secret
       gcloud secrets versions access latest --secret="${SECRET%.*}" > "${KOKORO_GFILE_DIR}/secrets/$SECRET"
       # execute secret file contents
       if [[ "$SECRET" != *json ]]; then
+        chmod u+x "${KOKORO_GFILE_DIR}/secrets/$SECRET"
```

Comment:
It's run_tests.sh permissions that were changed (https://stackoverflow.com/questions/1257592/how-do-i-remove-files-saying-old-mode-100755-new-mode-100644-from-unstaged-cha), so this shouldn't be needed

Reply:
removed the the chmod line